tofu/rancher/custom_cluster/README.md

```diff
@@ -0,0 +1,41 @@
+# Downstream Rancher Cluster rke2/k3s module
+
+This module deploys a downstream cluster on your rancher setup
+
+## Prerequisites
+
+* An api_key from your rancher setup
+* tofu installed on your client machine
+
+## Usage
+
+
+1.  **Create the downstream cluster**
+    * see the [variables section](#sample) to configure the cluster
+    ```bash
+    tofu -chdir=tofu/rancher/custom_cluster apply -auto-approve -var-file=/path/to/vars.tfvars
+    ```
+
+    Create a `vars.tfvars` file or use the `-var` flag to provide values for the variables defined in `variables.tf`.
+
+2.  **Destroy the downstream cluster:**
+
+    ```bash
+    tofu -chdir=tofu/rancher/custom_cluster destroy -auto-approve -var-file=/path/to/vars.tfvars
+    ```
+
+    Use the same `vars.tfvars` file or `-var` flags used during `apply`.
+
+## Outputs
+Refer to [outputs.tf](./outputs.tf) for a list of exported values.
```

Comment:
nit: Since there's only one exported value, it might be nice to put it here into more "human-readable" terms